packages/buckets/src/api/index.ts

```diff
@@ -1,4 +1,5 @@
 import { grpc } from '@improbable-eng/grpc-web'
+import { Repeater } from '@repeaterjs/repeater'
```

Comment:
This is a _much_ nicer lib for dealing with "pushable" async iterables. It is cleaner, provides lazy evaluation, and the API feels much more like the Promise-style API for creating new promises.